src/stylesheets/components/_popup.scss

```diff
@@ -174,6 +174,11 @@
 }
 
 // ============================
+
+.usa-tooltip {
+  text-align: right !important;
+}
```

Comment:
Could you confirm with @davereed on this? I see that the google docs also has align to right for tooltips, but I'm not sure if he meant for tooltip text content. Alignment to right seems a bit off for text. I don't think we want this style of behavior:
<img width="235" alt="Screen Shot 2022-01-26 at 3 54 24 PM" src="https://user-images.githubusercontent.com/73653244/151245370-4c0d7f6d-32e4-4765-8ba5-2734eecfbb22.png">

Reply:
removed the class for right align as its not needed